src/reformatters/noaa/hrrr/forecast_48_hour/template_config.py

```diff
@@ -119,7 +102,10 @@ def coords(self) -> Sequence[Coordinate]:
         dim_coords = self.dimension_coordinates()
         append_dim_coordinate_chunk_size = self.append_dim_coordinate_chunk_size()
 
+        spatial_coords = super().coords
```

Comment:
If we were going the Mixin route, maybe I'd call `self.spatial_coords` rather than trying to reason about why `super().coords` only has spatial coordinates.